File: ts-server/src/entity/Organization.ts
import { Entity, Column, PrimaryGeneratedColumn, OneToMany } from 'typeorm';
import { Category } from './Category';
import { Member } from './Member';
@Entity()
export class Organization {
    @PrimaryGeneratedColumn()
    id!: number;

    @Column({ type: "varchar", length: 50 })
    title!: string;

    @OneToMany(() => Category, category => category.organization)
    category!: Category[];

    @OneToMany(() => Member, member => member.organization)
    member!: Member[];
}

File: spring-server/src/main/java/com/mooc/entity/Organization.java
package com.mooc.entity;

import java.util.ArrayList;
import java.util.List;

import javax.persistence.CascadeType;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import javax.persistence.Table;

import lombok.Getter;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "Organization")
@Getter
@NoArgsConstructor
public class Organization {
	@Id
	@Column(name = "ORGANIZATION_ID")
	private String id;

	@OneToMany(mappedBy = "organization", cascade = { CascadeType.ALL })
	private List<Category> categoryList = new ArrayList<>();

	@OneToMany(mappedBy = "organization", cascade = { CascadeType.ALL })
	private List<Member> memberList = new ArrayList<>();

	public Organization(String id) {
		this.id = id;
	}
}
